apitools/base/py/transfer.py

```diff
@@ -483,13 +487,6 @@ def GetRange(self, start, end=None, additional_headers=None,
             response = self.__ProcessResponse(response)
             progress += response.length
             if response.length == 0:
-                if response.status_code == http_client.OK:
```

Comment:
I'm not super sure why this was removed. It feels like it should be okay regardless of version? Even if http_client.OK doesn't exist in some version, it would resolve to None and fall through to the raise exception below right? Maybe this was here for some debugging purpose?